src/sentry/status_checks/base.py

```diff
@@ -2,13 +2,23 @@
 
 import six
 
-from functools import total_ordering
-
 from sentry.utils.compat import implements_to_string
 
 
+def sort_by_severity(problems):
+    """\
+    Sort an iterable of ``Problem``s by their severity, from most severe to least severe.
+    """
+    return sorted(
+        problems,
+        key=lambda i: (
+            -Problem.SEVERITY_LEVELS[i.severity],
+            i.message,
+        ),
+    )
+
+
 @implements_to_string
-@total_ordering
```

Comment:
Removed this in favor of `sort_by_severity` since the combination of descending and ascending sort criteria doesn't exactly lend itself to a generalized total ordering.